### DISCUSSION TITLE
Request to Add Support for Vara Network on Wallet Connect

### DISCUSSION THREAD

**Discussion by pogorelov-aa** [WalletConnect/walletconnect-monorepo#5177] [Developer Support]
Dear Developers Team

I am writing to inquire if it would be possible for your team to add support for the Vara Network (non-EVM) to your Wallet Connect service. This addition would greatly enhance the versatility and user experience of your platform.

Thank you for considering this request. I look forward to your positive response.

Best regards

**Comment by glitch-txs** [comment]
Hello, the protocol is Chain Agnostic so any chain can part of it. 

Please check https://docs.walletconnect.com/cloud/chains/overview
